openedx/core/lib/xblock_utils.py

```diff
@@ -240,7 +240,7 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
         else:
             return frag
 
-    if isinstance(block, SequenceModule):
+    if isinstance(block, SequenceModule) or getattr(block, 'HIDDEN', False):
```

Comment:
OK, let's leave it out, then.
On Jul 1, 2015 7:14 AM, "Usman Khalid" <notifications@github.com> wrote:

> In openedx/core/lib/xblock_utils.py
> <https://github.com/edx/edx-platform/pull/8704#discussion_r33668843>:
>
> > @@ -240,7 +240,7 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, bloc
> >          else:
> >              return frag
> >
> > -    if isinstance(block, SequenceModule):
> > +    if isinstance(block, SequenceModule) or getattr(block, 'HIDDEN', False):
>
> One other complication leaving staff debug info in creates is that then we
> will need the bottom margin of the blocks div when a staff member is
> viewing but not when a student is viewing. Currently, if it is the
> HiddenModule, we do not show the bottom margin at all, since we assume
> there is not going to be any html.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8704/files#r33668843>.
>